src/webserver/WsBase.h

```diff
@@ -374,8 +374,8 @@ class WsSession : public WsBase,
                 // if the result is forwarded - just use it as is
                 // but keep all default fields in the response too.
                 if (isForwarded)
-                    for (auto const& [k, v] : result)
-                        response.insert_or_assign(k, v);
+                    for (auto const& [k, v2] : result)
```

Comment:
The original v can be more meaningful
`auto [response, timeDiff] =
                util::timed([&]() { return RPC::buildResponse(*context); });`